config/infection.json.dist

```diff
@@ -21,6 +21,48 @@
         }
     },
     "timeout": 10,
-    "minMsi": 98,
-    "minCoveredMsi": 100
+    "minMsi": 100,
+    "minCoveredMsi": 100,
+    "mutators": {
+        "@default": true,
+        "Break_": {
+            "ignore": [
+                "Psl\\Collection\\*Map::zip"
```

Comment:
When the break gets transformed into a continue, it will produce the same output - yet slower.
So this is a false positive:

https://github.com/azjezz/psl/blob/4affb62545080af0457f78a2bdcfc33cc4057f47/src/Psl/Collection/Map.php#L378-L380